### ISSUE TITLE
Anno1800 Game Update 15.1 (19th Oct 2022)

### ISSUE THREAD

**Issue by greycubesgav** [xforce/anno1800-mod-loader#170]
Hi Anno just updated to 15.1, so receiving the Version is not compatible warning.

![Screenshot 2022-10-23 130218](https://user-images.githubusercontent.com/3426859/197386036-17fc9167-7010-4631-b990-6c05944ae464.png)

Thanks
Gavin

**Comment by jjalexander** [comment]
I don't get this message, but the game never starts.

**Comment by nuqakaniqan** [comment]
same error here, strangely it was working one time for me with version 15.1....

**Comment by greycubesgav** [comment]
> same error here, strangely it was working one time for me with version 15.1....

Yes it was the same for me. It was only after coming back to Anno a day later that it stopped working. Maybe it needed a reboot or there was another silent update in the background?

I'm running Anno 1800 under the Ubisoft Launcher.

**Comment by xforce** [comment]
Does this still happen with Version 0.9.1 (released yesterday)?

**Comment by Cherrywickle** [comment]
Just yesterday it was working fine with 395 mods. no problems unless i raised the graphics settings too high.

**Comment by Cherrywickle** [comment]
Its working now, just used a previous version 0.9.0

**Comment by Morgrain** [comment]
My game has also started randomly crashing (without error message, it just drops to desktop and the game disappears) ever since the game updated/ this mod loader updated.

**Comment by xforce** [comment]
> My game has also started randomly crashing (without error message, it just drops to desktop and the game disappears) ever since the game updated/ this mod loader updated.
> 
> On top of that, ALL auto saves when playing with the 0.9.1 are now corrupted, and can't be loaded (not even with the 0.9.0).
> 
> I had to find the last manual save with 0.9.0, and loaded from that - with 0.9.0. Now my save works properly again. You somehow broke the game with 0.9.1.
> 
> Edit: well, the game still crashes with the 0.9.0 - even though it takes much longer (0.9.1 = 10-20 minutes, 0.9.0 = 40-50 minutes). Odd.
> 
> Edit 2: the problem was that ALL save files that were touched by the 0.9.1 are corrupted, I loaded an old savefile - from 2 days ago - before I played with 0.9.1 and this works properly now (when using the old 0.9.0 !)
> 
> 0.9.1 is definitely broken and is corruption savegames.
> 
> The update should be pulled before more people are affected.
> 
> WARNING: DO NOT USE!

Can't reproduce on my machine. I have several very old auto-saves that work perfectly fine for me

**Comment by tefracky** [comment]
For me, the game stopped loading using 0.9.1. (Anno crashed while loading). Using 0.9.0, all works fine.

**Comment by greycubesgav** [comment]
Just to confirm and to close off the initial reported issue:
 ✅ I can confirm **Version 0.9.1** of the mod-loader works with my current Anno install.

* Anno1800.exe File Version: 17.52448.1.34411

![Screenshot 2022-10-28 140611](https://user-images.githubusercontent.com/3426859/198573037-48edfe76-305b-4b85-ab50-11970d2a7fd4.png)

* Anno game version shown on boot: 15.1.1166560

![Screenshot 2022-10-28 140802](https://user-images.githubusercontent.com/3426859/198573502-cb428f1f-878c-465e-baa6-7191691cf431.png)

**Comment by greycubesgav** [comment]
I would suggest anyone experiencing corrupted saves to raise a new ticket so the discussion can be focused there.
Note: I have not seen any saved game corruption issues.
